sdk/JavaScript/TestFramework/js/testFramework.js

```diff
@@ -36,15 +36,15 @@ function createZumoNamespace() {
         this.canRunUnattended = true;
         this.logs = [];
         this.requiredFeatures = [];
-        if (requiredRuntimeFeatures) {
-            if (typeof requiredRuntimeFeatures === 'string') {
-                requiredRuntimeFeatures = [requiredRuntimeFeatures];
+        if (requiredRuntimeFeatures) {
```

Comment:
yes, but then I can't easily add line comments.  I'm ok with changing tabs -> whitespace, but make sure we aren't changing line endings unless on purpose.